src/Numerics/Mesh/Grids.jl

```diff
@@ -167,27 +173,34 @@ struct DiscontinuousSpectralElementGrid{
     "Array indicating if a degree of freedom (real or ghost) is active"
     activedofs
 
-    "1-D lvl weights on the device"
+    "1-D lvl weights on the device (one for each dimension)"
```

Comment:
Is `lvl` an acronym used elsewhere? I might have missed it

Reply:
Thanks. Should have been lgl